docs/tables/country.md

```diff
@@ -0,0 +1,12 @@
+# Country
+The country table is made by COKI, it contains country data, including each country's: ISO 3166 name, Wikipedia name,
+ISO 3166 two letter alpha2 code, ISO 3166 three letter alpha3 code, region name, subregion name, Wikidata QID and
+Wikipedia URL.
```

Comment:
Would it be possible to add the code or more details on where we get these values from?
E.g. if this table accidentally gets deleted, how can we recreate it?

Reply:
It is just a CSV file that I manually created. 

I've added a CSV and JSON Lines version in the [academic_observatory_workflows/fixtures/doi](https://github.com/The-Academic-Observatory/academic-observatory-workflows/tree/feature/countries_and_regions/academic_observatory_workflows/fixtures/doi) folder and it is also loaded in BigQuery.

I've also specified a bit more information about where the region and subregion names come from.

Reply:
Nice, thanks!